externs/shaka/manifest.js

```diff
@@ -203,6 +203,7 @@ shaka.extern.ServiceDescription;
  * @typedef {{
  *   keySystem: string,
  *   encryptionScheme: string,
+ *   keySystemUris: (Set.<string>|undefined),
```

Comment:
This was made a Set to match keyIds. I originally named it `keySystemUris` to be more generic, since DrmInfo is used both for hls/fairplay and DASH, but its type is possibly undefined so that the property could be unavailable when the data isn't present.